study-builder/studies/rarex/i18n/en.json

```diff
@@ -250,10 +250,10 @@
   "general_information_patient_country_birthplace": "Country/Territory where the patient was born*",
   "general_information_patient_country_birthplace_req_hint": "Country/Territory cannot be empty",
 
-  "general_information_patient_city_birthplace": "Name of the city/municipality where the patient was born*",
+  "general_information_patient_city_birthplace": "City/Municipality where the patient was born*",
   "general_information_patient_city_birthplace_req_hint": "City/Municipality cannot be empty",
 
-  "general_information_patient_state_birthplace": "State where the patient was born*",
+  "general_information_patient_state_birthplace": "State/Province where the patient was born*",
```

Comment:
This is what I'm seeing in ticket:
> Change `State where the patient was born` to `State/Province where the patient was born (if State/Province is not applicable to the birthplace, enter any other geographic descriptor (e.g. Republic, etc.)`

Reply:
The way I am reading ticket is that there was a hope that we could do something smart if "State/Province" was not applicable based on the Country/Territory entered earlier in form.
Unfortunately, since country/territory is just a string (and our list of countries does not really contain information about whether national subdivisions are state, provinces) this not something that is in our power to do.
I will update ticket to make David O aware of what we doing here.